Resources/Prototypes/Nyanotrasen/Entities/Objects/Weapons/Melee/blunt.yml

```diff
@@ -8,7 +8,8 @@
     sprite: Nyanotrasen/Objects/Weapons/Melee/kanabou.rsi
     state: icon
   - type: MeleeWeapon
```

Comment:
Is the size of the kanabou still 5? That should probably be changed unless there's a good reason it can be stored in a pocket.

Reply:
yeah this really should not fit in a pocket at all, changed the size to 20 so it at least fits in backpacks.